src/content/language/dot-shorthands.md

```diff
@@ -0,0 +1,324 @@
+---
+title: Dot shorthands
+description: Learn about the dot shorthand syntax in Dart.
+prevpage:
+  url: /language/enums
+  title: Enums
+nextpage:
+  url: /language/extension-methods
+  title: Extension methods
+---
+
+<?code-excerpt replace="/ *\/\/\s+ignore_for_file:[^\n]+\n//g; /(^|\n) *\/\/\s+ignore:[^\n]+\n/$1/g; /(\n[^\n]+) *\/\/\s+ignore:[^\n]+\n/$1\n/g"?>
+
+:::experimental
+Support for dot shorthand syntax is experimental and
+can only be used on the `main` channel.
+:::
+
+## Grammar
+
+Dot shorthand syntax `.foo` lets you write more concise Dart 
+code by omitting the type when the compiler can infer it 
+from context. This provides a clean alternative to writing 
+the full `ContextType.foo` when accessing enum values, 
+static members, or constructors.
+
+In essence, the dot shorthands grammar allows an expression 
```

Comment:
Does it make sense to just say "dot shorthands" here, dropping the "grammar"? We don't mention the lower-level concept of a grammar a lot (or at all maybe) in the `/language` docs.

```suggestion
In essence, dot shorthands allow an expression 
```